packages/vscode-ui5-language-assistant/tools/run-update.ts

```diff
@@ -0,0 +1,67 @@
+import axios from "axios";
+import fs from "fs/promises";
+import { join } from "path";
+import prettier from "prettier";
+
+export const BASE_PATH = join("src", "manifest");
```

Comment:
can we please apply absolute path instead of relative?

Reply:
added